typegraph/core/src/runtimes/prisma/type_generation/additional_filters.rs

```diff
@@ -24,7 +24,7 @@ pub struct Skip;
 
 impl TypeGen for Skip {
     fn generate(&self, _context: &PrismaContext) -> Result<TypeId> {
-        t::integer().x_min(0).named(self.name()).build()
```

Comment:
x_min(-1) or is there a non-exclusive flag?

Reply:
`min(0)` would work.